### PR TITLE
v2.0.0

### PR DESCRIPTION
**Pull request by GRA0007**
- Separate build command

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: main
+
+on: [push]
```
Comment (giraugh) [inline .github/workflows/main.yml:3]: do we want this to run on any push? or just a specific branch?

Comment (giraugh) [reply .github/workflows/main.yml:3]: we might want
```yml
on:
  push:
    branches:
      - main
```

Comment (giraugh) [reply .github/workflows/main.yml:3]: oh wait up, do we want to attempt to build on any push but publish only on pushes to main?

Comment (GRA0007) [reply .github/workflows/main.yml:3]: Nah cause we can put a test command into that build step (#11), and the publish step only runs on main

**Review by giraugh — APPROVED** [review]
LGTM :)

**Pull request merged**